### PR TITLE
Drone 0.5 support

### PR DESCRIPTION
**Pull request by jmccann**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Docker image for the Drone Rubygems plugin
```
Comment (tboerger) [inline Dockerfile]: I would drop these comments entirely

File: README.md [hunk 2]
```diff
@@ -1,9 +1,5 @@
 # drone-rubygems
 
-[![Build Status](http://beta.drone.io/api/badges/drone-plugins/drone-rubygems/status.svg)](http://beta.drone.io/drone-plugins/drone-rubygems)
```
Comment (tboerger) [inline README.md:3]: No build status and so on?

File: README.md [hunk 3]
```diff
@@ -14,88 +10,15 @@ Build the binary using `make`:
...
-        "name": "my_awesome_tool"
-    }
-}
-EOF
+docker run --rm \
+  -e RUBYGEMS_API_KEY=abcd12345 \
+  -e PLUGIN_REPO=octocat/hello-world \
+  -v $(pwd):$(pwd) \
+  -w $(pwd) \
+  plugins/drone-rubygems --help
```
Comment (tboerger) [inline README.md]: Wrong org

File: main.go [hunk 4]
```diff
@@ -3,103 +3,94 @@ package main
...
-	plugin.Param("build", &build)
-	plugin.Param("vargs", &vargs)
-	plugin.MustParse()
-
-	if (len(vargs.Username) == 0 || len(vargs.Password) == 0) && len(vargs.APIKey) == 0 {
-		fmt.Println("Please provide an API key or username/password credentials")
-
-		os.Exit(1)
-		return
+	logrus.Info(fmt.Printf("Drone Rubygems Plugin built from %s", buildCommit))
```
Comment (tboerger) [inline main.go:14]: For the official plugins we have dropped that

Comment (jmccann) [reply main.go:14]: I think I'm going to leave this for now.  I like knowing the commit a plugin was built from for debug purposes.

File: README.md (hunk 2 above)
Comment (jmccann) [reply README.md:3]: Added back build status badge

**Pull request merged**